### PR TITLE
Camera code final commit

### PR DESCRIPTION
**Pull request by sjc625**
Code is functioning, and C code fits all requirements. 
Todo: update readme.

### CODE REVIEW

File: camextra/main.c [hunk 1]
```diff
@@ -0,0 +1,329 @@
+#include <stdio.h>
+#include <unistd.h>
+#include <stdint.h>
+#include <fcntl.h>
+#include <termios.h>
+#include <stdlib.h>
+#include <string.h>
+
+//char _SYNC_FINAL_COMMAND[6] = {0xAA, 0x0E, 0x00, 0x00, 0xF5, 0x00};
```
Comment (JoshuaHassler) [inline camextra/main.c]: remove these if you are not going to use them :)

File: ccam/Makefile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+main:
+	gcc ucam.c -o ucam
+clean:
```
Comment (JoshuaHassler) [inline ccam/Makefile]: clean should be a phony rule.
```suggestion
.PHONY: clean
clean:
```

File: ccam/ucam.h [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+#include <string.h>
+#include <termios.h>
+#include <unistd.h>
+
+int camera_Sync(int stream, int debug);
+int camera_Initialize(int stream, int debug);
+int camera_Size(int stream, int debug);
+int camera_Jpg(int stream, char* str, int debug);
+
+int camera_Sync(int stream, int debug) {
```
Comment (JoshuaHassler) [inline ccam/ucam.h]: You should NOT put function definitions in a header file. It is bad practice and can confuse the compiler.

File: pycam/jpeg.py [hunk 4]
```diff
@@ -234,10 +234,10 @@ def camera_get_picture_jpg():
 		ser.write(ack)
 
 		received = ser.read(512)
-		size = received[4]
-		temp = received[3]
-		temp *= 256
-		size += temp
+		#size = received[4]
```
Comment (JoshuaHassler) [inline pycam/jpeg.py]: Just delete thing you don't need. If you want it again that's why we have git commit history :)

File: camextra/main.c [hunk 5]
```diff
@@ -0,0 +1,329 @@
+#include <stdio.h>
```
Comment (JoshuaHassler) [inline camextra/main.c]: If this file is unused, it can probably be removed

**Review by JoshuaHassler — COMMENTED** [review]
I know this is more of a proof of concept right now, but splitting things into multiple files more clearly could help with debugging. Also, remember to squash your commits before merging.

**Comment by sjc625** [comment]
did i fix it/ how do i fix it

**Review by JoshuaHassler — APPROVED** [review]
C code looks fine ( other than function definitions in header files :) )

**Pull request merged**